.gitignore

```diff
@@ -9,5 +9,14 @@
 __pycache__/
 /authentication/migrations/
 /profiles/migrations/
+/search/migrations/
+/db.sqlite3
+/.sqlite3
+/*.sqlite3
+/forum/local_settings.py
+/node_modules
+package-lock.json
+package.json
 /FrontEnd/node_modules
 /FrontEnd/package-lock.json
+/FrontEnd/package.json
```

Comment:
Same `package.json` shouldn't be in git ignore

Reply:
The same like previouse)